ARExperiment/ARExperiment/ARSession/StatusViewController.swift

```diff
@@ -0,0 +1,74 @@
+/*
+See LICENSE folder for this sample’s licensing information.
+
+Abstract:
+Utility view controller for showing messages above the AR view.
+*/
+
+import Foundation
+import ARKit
+
+class StatusViewController: UIViewController {
```

Comment:
I don't think we need the comments in this file. Many of them (e.g. `restartExperienceHandler `) are obvious enough that we don't need the comments.

Reply:
(I know this was a copy and paste, but we should make it our standards)

Reply:
I left the comments because this file was created by Apple, is from their examples code that basically shows the user the status of the session. It felt wrong to just delete the comments and "make it ours" but if you think is better if we do it then I will.

Reply:
well, we're now responsible for maintaining this, right? if we're copy and pasting it. So we should tidy it up as best as we can, because else it may become technical debt we're just pretending we're not maintaining